lsp-semantic-tokens.el

```diff
@@ -253,17 +254,72 @@ Faces to use for semantic token modifiers if
         (tokenTypes . ,(apply 'vector (mapcar #'car lsp-semantic-token-faces)))
         (formats . ["relative"])))))
 
+(defvar lsp--semantic-tokens-pending-full-token-requests '()
+  "Buffers which should have their semantic tokens refreshed on idle.
+
+This is an alist of the form ((buffer_i . fontify_immediately_i) ...); entries with
+fontify_immediately set to t will immediately refontify once their token
+request is answered.")
```

Comment:
To fix CI
```suggestion
This is an alist of the form ((buffer_i . fontify_immediately_i) ...); entries 
with fontify_immediately set to t will immediately refontify once their 
token request is answered.")
```

Reply:
thanks! Pushed those changes, and addressed a few similar complaints raised by the byte compiler